### PR TITLE
[ROC_BERT] Make CI happy

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

Fixes a small slow test that was failing when trying to play with 8-bit conversion for BERT models

cc @ydshieh

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
LGTM thanks

**Comment by ydshieh** [comment]
Hi @younesbelkada Thank you for trying make CI happy.

But I am not sure why we need `torch.allclose` for 2 integer tensors. This should be used for float tensors. For the integer outputs (here token ids), they should be equal, so `self.assertEqual` would be the one to use I think.

**Comment by younesbelkada** [comment]
Thanks @ydshieh for the heads up, indeed I believe we can use that. Let me update it

File: tests/models/roc_bert/test_modeling_roc_bert.py [hunk 1]
```diff
@@ -705,4 +705,4 @@ def test_inference_masked_lm(self):
         # convert to tokens is: ['[CLS]', '巴', '*', '黎', '是', '法', '国', '的', '首', '都', '[SEP]']
         expected_output = torch.tensor([[101, 2349, 115, 7944, 3221, 3791, 1744, 4638, 7674, 6963, 102]])
 
-        self.assertTrue(output_ids, expected_output)
+        self.assertTrue(torch.allclose(output_ids, expected_output))
```
Comment (younesbelkada) [inline tests/models/roc_bert/test_modeling_roc_bert.py:708]:
```suggestion
        self.assertEqual(output_ids, expected_output)
```

**Review by ydshieh — APPROVED** [review]
Thank you @younesbelkada for the `egalité`

**Review by sgugger — APPROVED** [review]

File: tests/models/roc_bert/test_modeling_roc_bert.py (hunk 1 above)
Comment (ArthurZucker) [reply tests/models/roc_bert/test_modeling_roc_bert.py]: So younes was right : 
```python
>>> self.assertEqual(output_ids, expected_output)
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/conda/envs/py39/lib/python3.9/unittest/case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/opt/conda/envs/py39/lib/python3.9/unittest/case.py", line 827, in _baseAssertEqual
    if not first == second:
RuntimeError: Boolean value of Tensor with more than one value is ambiguous
```

Opening up a [PR to fix this ](https://github.com/huggingface/transformers/pull/20469)

**Pull request merged**